frontend/src/metabase/admin/settings/components/widgets/AdminSettingInput.tsx

```diff
@@ -145,16 +157,18 @@ export function BasicAdminSettingInput({
           id={name}
           checked={localValue}
           onChange={(e) => handleChange(e.target.checked)}
-          label={localValue ? t`Enabled` : t`Disabled`}
+          label={switchLabel ?? (localValue ? t`Enabled` : t`Disabled`)}
           w="auto"
         />
       );
     case "radio":
       return (
         <Radio.Group id={name} value={localValue} onChange={handleChange}>
-          {options?.map(({ label, value }) => (
-            <Radio key={value} value={value} label={label} />
-          ))}
+          <Stack gap="sm">
+            {options?.map(({ label, value }) => (
+              <Radio key={value} value={value} label={label} />
+            ))}
+          </Stack>
```

Comment:
without the stack, these are too squished together

Reply:
Great minds think alike! (I did the same in my branch)